controllers/gatekeeper_controller.go

```diff
@@ -238,13 +244,30 @@ func (r *GatekeeperReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	if err := r.handleCPSController(ctx, gatekeeper); err != nil {
 		if errors.Is(err, errCrdNotReady) {
 			// ConstraintPodStatus CRD is not ready, wait for the CRD is ready
-			return ctrl.Result{RequeueAfter: time.Second * 3}, nil
+			requeueTime = time.Second * 3
 		} else {
-			return ctrl.Result{}, err
+			resultErr = err
 		}
 	}
 
-	return ctrl.Result{}, nil
+	if err := r.handleConfigController(ctx); err != nil {
+		if errors.Is(err, errCrdNotReady) {
+			// Config CRD is not ready, wait for the CRD is ready
+			requeueTime = time.Second * 3
+		} else {
+			resultErr = err
+		}
+	}
+
+	// Create a Config although CPS and Config controllers are failed
+	err = r.initConfig(ctx, gatekeeper)
```

Comment:
Won't this conflict with the new controller launched by `handleConfigController`? I imagine they will both try to update it at the same time.

Reply:
Thhe initConfig will create the config and add the exempt namespaces. It won't fall into `setExemptNamespaces` when it creates initConfig

Reply:
I'm saying that the config controller will reconcile at the same time this is called if the config already exists.

I think `initConfig` should likely only create the `Config` object if it doesn't exist and then let the config controller handle setting the correct values since creating the `Config` object here would trigger the config controller to reconcile.